backend/apps/slack/services/message_auto_reply.py

```
@@ -45,9 +46,19 @@ def generate_ai_reply_if_unanswered(message_id: int):
         thread_ts=message.slack_message_id,
     )
+
+    # Log the interaction so reaction_added can match 👍/👎 back to this reply.
+    reply_ts = response.get("ts", "") if response else ""
+    BotInteraction.objects.create(
+        channel_id=message.conversation.slack_channel_id,
+        user_id=message.raw_data.get("user", ""),
+        user_message=message.text,
+        bot_response=ai_response_text,
+        slack_reply_ts=reply_ts,
+    )
```

Ruff (0.15.18)

[warning] No newline at end of file

Add trailing newline

(W292)